pages.fr/linux/pacman.md

```diff
@@ -1,6 +1,7 @@
 # pacman
 
-> Outil de gestion de paquets sur Arch Linux. 
+> Outil de gestion de paquets sur Arch Linux.
+> Plus d'informations : <https://man.archlinux.org/man/pacman.8.en>.
```

Comment:
Slightly shortening this to remove the language code works too. Maybe then it'll display in the user's native language?
```suggestion
> Plus d'informations : <https://man.archlinux.org/man/pacman.8>.
```

Same goes for the other URLs.

Reply:
@sbrl links like https://man.archlinux.org/man/pacman.8.fr redirect to the `en` version 🤷

Reply:
But removing the code is a good idea still, to be as language agnostic as possible.